scripts/generate-config.js

```diff
@@ -53,7 +53,7 @@ const preamble = `
 version: 2.1
 
 orbs:
-    node: circleci/node@5.0.0
+    node: circleci/node@1.1
```

Comment:
@BlueWinds I noticed a handful of changes in this config file unrelated to the new included image. What was the reasoning behind these? Just trying to make sure we have context as we manage this moving forward. Thanks.

Reply:
https://github.com/cypress-io/cypress-docker-images/pull/595#discussion_r807312759